ios/Classes/BetterPlayer.m

```diff
@@ -856,7 +856,7 @@ - (void) initLimitedBlackCoverView {
 }
 
 - (void) showLimitedBlackCoverView {
-    if (self._betterPlayerView == nil) {
+    if (self._betterPlayerView.superview == nil) {
```

Comment:
memo:

I think, We had to fix this at that time.
https://github.com/dwango-nfc/betterplayer/pull/71/files#diff-4b6db25e7cbbc0e1b666cb708e68032d43c6e1ca72216f387e89d0dfc254ca3eR863